requirements.txt

```diff
@@ -10,3 +10,4 @@ numpy
 Unidecode>=1.0.22
 beautifulsoup4
 nltk
+tensorboardX
```

Comment:
Actually, could we move `tensorboardX` to extra, and only import it when doing logging? It seems like it shouldn't be necessary unless we are actually doing tensorboard logging.

Reply:
tensorboard logging is currently done by default and there's no option to disable it. We could add an option, but I thought that would only make things more complicated, and people might decide to want to look at the visualization after they already started training. It's also not expensive to do the logging. What do you think?

Reply:
OK, that's fine I guess. I'm just wary of introducing more and more dependencies, which will break at some point when the package version changes, etc.

That being said, Tensorboard is a feature that I think a lot of people would want, so in this case it's probably fine.